src/modules/assertions/require_inline_coding.ts

```diff
@@ -0,0 +1,13 @@
+import { context } from "@actions/github";
+import _ from "lodash";
+
+export const  requireInlineCoding = () => {
+  const solfileRegex = /http:\/\/\S+.sol/g;
+  const prBodyString = JSON.stringify(context.payload?.pull_request?.body);
+  if (!(prBodyString) || (prBodyString.length === 0)) {return 0;}
+  if (solfileRegex.test(prBodyString)) {
+    console.log(`EIP-BOT is terminating. Please replace external link http://.../file.sol: ${prBodyString.match(solfileRegex)} for inline code reference..`);
+    process.exit(1);
+  }
```

Comment:
`String.match` in JavaScript returns an array, not a single string.  We should log each of the items separately before exiting in the case where there are multiple matches.

Something like this should work (note: only pseudocode, you'll need to compile and test this to verify it actually works):
```suggestion
  const matches = prBodyString.match(solfileRegex)
  for (let i = 0; i < matches.length; ++i) {
    console.log(`EIP-BOT is terminating. Please replace external link http://.../file.sol: ${prBodyString.match(solfileRegex)} for inline code reference..`);
  }
  if (matches.length > 0) process.exit(1)
```